molecule/default/tests/test_minio_default.py

```diff
@@ -0,0 +1,43 @@
+import os
+import yaml
+import pytest
+import testinfra.utils.ansible_runner
+
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+@pytest.fixture()
+def AnsibleDefaults(Ansible):
+    with open("../../defaults/main.yml", 'r') as stream:
+        return yaml.load(stream)
+
+
+@pytest.mark.parametrize('minio_bin_var', [
+    'minio_server_bin',
+    'minio_client_bin',
+])
+def test_minio_installed(host, AnsibleDefaults, minio_bin_var):
```

Comment:
Why not put this in `test_minio_common.py`?

Reply:
Because otherwise that would be tested in every scenario.

The idea is to test it once, in the default scenario, and eventually add the proper tests if we decide to override the default values for the `minio_server_bin` and `minio_client_bin` variables.

Reply:
I am not very fond of this solution, but it is ok :+1: